app/templates/_package.json

```diff
@@ -15,7 +15,8 @@
     "gulp-imagemin": "^2.2.1",
     "gulp-load-plugins": "^0.10.0",
     "gulp-minify-css": "^1.1.1",
-    "gulp-minify-html": "^1.0.0",<% if (includeSass) { %>
+    "gulp-minify-html": "^1.0.0",
+    "gulp-rev-all": "^0.8.0",<% if (includeSass) { %>
```

Comment:
Ok, I can work on that, but it'll require about 3 plugins to do what rev-all does. 
Interesting strategy to blacklist forks...